File: TFC-TourGuide-PHP/modelos/Anuncio.php
<?php

// Esta clase representa una Usuario

class Anuncio{
    
    // Variables de clase
    private $idturista,$ciudad,$idtipo,$nombre,$acompanantes,$fechacreacion,$fecha1,$fecha2,$mensaje,$estado;

    // Constructor
    public function __construct($idturista,$ciudad,$idtipo,$nombre,$acompanantes,$fechacreacion,$fecha1,$fecha2,$mensaje,$estado)
    {
        $this->idturista = $idturista;
        $this->ciudad = $ciudad;
        $this->idtipo = $idtipo;
        $this->nombre = $nombre;
        $this->acompanantes = $acompanantes;
        $this->fechacreacion = $fechacreacion;
        $this->fecha1 = $fecha1;
        $this->fecha2 = $fecha2;
        $this->mensaje = $mensaje;
        $this->estado = $estado;
    }

    public function getEstado(){
        return $this->estado;
    }

    public function getNombre(){
        return $this->nombre;
    }

    public function getIDTurista(){
        return $this->idturista;
    }

    public function getTipo(){
        return $this->idtipo;
    }

    public function getMensaje(){
        return $this->mensaje;
    }


    public function getFechaCreacion(){
        return $this->fechacreacion;
    }

    public function getCiudad(){
        return $this->ciudad;
    }

    public function getAcompanantes(){
        return $this->acompanantes;
    }
    
    public function getFecha1(){
        return $this->fecha1;
    }

    public function getFecha2(){
        return $this->fecha2;
    }

    // Muestra los datos de la Usuario
    public function toString()
    {
        return
            [
                "nombre" => utf8_encode($this->nombre),
                "ciudad" => utf8_encode($this->ciudad),
                "acompanante" => utf8_encode($this->acompanante),
                "fecha1" => utf8_encode($this->fecha1),
                "fecha2" => utf8_encode($this->fecha2)
            ];
    }
}

File: TFC-TourGuide-Android/app/src/main/java/com/example/tourguide/modelo/Anuncio.java
package com.example.tourguide.modelo;
import java.util.Date;

public class Anuncio {
    private String tokenTurista,nombre,ciudad,acompanantes,mensaje,id,provincia;
    private Date fecha1,fecha2;
    private Date fechaCreacion;
    private Intereses tipo;
    private Turista turista;
    private Estados estado;

    public Anuncio(String id, String tokenTurista, String nombre, String ciudad,String provincia, Intereses tipo, String acompanantes, Date fechaCreacion, Date fecha1, Date fecha2, String mensaje, Turista turista,Estados estado) {
        this.id = id;
        this.tokenTurista = tokenTurista;
        this.nombre = nombre;
        this.ciudad = ciudad;
        this.provincia = provincia;
        this.tipo = tipo;
        this.acompanantes = acompanantes;
        this.fechaCreacion = fechaCreacion;
        this.fecha1 = fecha1;
        this.fecha2 = fecha2;
        this.mensaje = mensaje;
        this.turista = turista;
        this.estado = estado;
    }

    public Estados getEstado() {
        return estado;
    }

    public void setEstado(Estados estado) {
        this.estado = estado;
    }

    public String getProvincia() {
        return provincia;
    }

    public void setProvincia(String provincia) {
        this.provincia = provincia;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getMensaje() {
        return mensaje;
    }

    public Turista getTurista() {
        return turista;
    }


    public void setTurista(Turista turista) {
        this.turista = turista;
    }

    public void setMensaje(String mensaje) {
        this.mensaje = mensaje;
    }

    public String getTokenTurista() {
        return tokenTurista;
    }

    public void setTokenTurista(String tokenTurista) {
        this.tokenTurista = tokenTurista;
    }

    public Date getFechaCreacion() {
        return fechaCreacion;
    }

    public void setFechaCreacion(Date fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }

    public Intereses getTipo() {
        return tipo;
    }

    public void setTipo(Intereses tipo) {
        this.tipo = tipo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCiudad() {
        return ciudad;
    }

    public void setCiudad(String ciudad) {
        this.ciudad = ciudad;
    }

    public String getAcompanantes() {
        return acompanantes;
    }

    public void setAcompanantes(String acompanantes) {
        this.acompanantes = acompanantes;
    }

    public Date getFecha1() {
        return fecha1;
    }

    public void setFecha1(Date fecha1) {
        this.fecha1 = fecha1;
    }

    public Date getFecha2() {
        return fecha2;
    }

    public void setFecha2(Date fecha2) {
        this.fecha2 = fecha2;
    }

    @Override
    public String toString() {
        return "Anuncio{" +
                "tokenTurista='" + tokenTurista + '\'' +
                ", nombre='" + nombre + '\'' +
                ", ciudad='" + ciudad + '\'' +
                ", acompanantes='" + acompanantes + '\'' +
                ", mensaje='" + mensaje + '\'' +
                ", id='" + id + '\'' +
                ", provincia='" + provincia + '\'' +
                ", fecha1=" + fecha1 +
                ", fecha2=" + fecha2 +
                ", fechaCreacion=" + fechaCreacion +
                ", tipo=" + tipo +
                ", turista=" + turista +
                ", estado=" + estado +
                '}';
    }
}
